libs/execution_context/ebpf_program.c

```diff
@@ -465,6 +532,25 @@ ebpf_program_load_byte_code(ebpf_program_t* program, ebpf_instruction_t* instruc
     return return_value;
 }
 
+ebpf_result_t
+ebpf_program_load_code(
+    _Inout_ ebpf_program_t* program, ebpf_code_type_t code_type, _In_ const uint8_t* code, size_t code_size)
+{
+    ebpf_result_t result = EBPF_SUCCESS;
+    program->parameters.code_type = code_type;
+    if (program->parameters.code_type == EBPF_CODE_NATIVE)
+        result = _ebpf_program_load_machine_code(program, code, code_size);
+    else if (program->parameters.code_type == EBPF_CODE_EBPF)
+        result = _ebpf_program_load_byte_code(
+            program, (const ebpf_instruction_t*)code, code_size / sizeof(ebpf_instruction_t));
+    else {
+        result = EBPF_INVALID_ARGUMENT;
+        goto Exit;
+    }
+Exit:
```

Comment:
```suggestion
    else
        result = EBPF_INVALID_ARGUMENT;
```
nit: make 542, 544, and 547 be consistent